src/core/Search/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: '10.18rem',
+    color: theme.palette.text.hint,
+    '&:before': {
+      borderBottom: `0.0625rem solid ${theme.palette.border.main}`,
+    },
+    '& input': {
+      '&:focus': {
+        color: theme.palette.text.primary,
+      },
+    },
```

Comment:
```suggestion
    '& input': {
      '&:focus': {
        color: theme.palette.text.primary,
        borderColor: theme.palette.highlight
      },
    },
```

Reply:
What I think MUI is doing is inferring the color from `primary.main` which in this case is same for litmus and other platforms, but for platforms like `Kubera Portal` and `Kubera Core` the `primary main` and `highlight` are different, so they won't match as you can also see in your screenshots, you'd need to mention `borderColor` on &input as well to resolve this issue.